### PR TITLE
Fix header logo and icons

### PR DESCRIPTION
**Pull request by skjnldsv**
Only invert if the logo is the default
Also fixed the icons sizes
| Before | After |
|:---------:|:------:|
|Capture d’écran_2022-04-28_11-31-06|Capture d’écran_2022-04-28_11-30-19|

Peek 28-04-2022 09-44

https://github.com/nextcloud/server/issues/32072

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]
Works for the logo now.

The more button on the app list still has the wrong color thou:
Bildschirmfoto von 2022-04-28 10-06-50

**Comment by skjnldsv** [comment]
> The more button on the app list still has the wrong color thou: [Bildschirmfoto von 2022-04-28 10-06-50](https://user-images.githubusercontent.com/213943/165707176-1cdc51b8-0708-4d11-a7e1-3cee1e9ed057.png)

fixed

File: core/css/header.scss [hunk 1]
```diff
@@ -182,12 +182,8 @@
 		left: 12px;
 		top: 1px;
 		bottom: 1px;
-
-		img {
-			opacity: 0;
-			max-width: 100%;
-			max-height: 200px;
-		}
+		// Invert custom logo if primary too bright
```
Comment (come-nc) [inline core/css/header.scss:185]: non-custom, I think?

**Review by come-nc — APPROVED** [review]

**Comment by skjnldsv** [comment]
Fixed tests

**Comment by skjnldsv** [comment]
Fixed tests

**Review by Pytal — APPROVED** [review]

**Pull request merged**